src/map/clif.cpp

```diff
@@ -16875,8 +16874,7 @@ void clif_quest_send_mission(struct map_session_data *sd)
 void clif_quest_add(struct map_session_data *sd, struct quest *qd)
 {
 	int fd = sd->fd;
-	int i, offset;
-	struct quest_db *qi = quest_search(qd->quest_id);
+	auto qi = quest_search(qd->quest_id);
```

Comment:
„auto“ is reducing readability here.
The type is not redundant here because:
- First use of „quest_search“ in this function.
- Type is not immediately obvious.